activiti-core/activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/model/ProcessVariablesMapping.java

```diff
@@ -17,13 +17,12 @@
 
 package org.activiti.spring.process.model;
 
-import java.util.HashMap;
 import java.util.Map;
 
 public class ProcessVariablesMapping {
 
-    private Map<String, Mapping> inputs = new HashMap<>();
-    private Map<String, Mapping> outputs = new HashMap<>();
+    private Map<String, Mapping> inputs;
```

Comment:
if we initialize these maps by default, this use case:
```
"mappings": {
"UserTask_1negxpm": { }
}
```
and this one
```
       "mappings": {
           "UserTask_1negxpm": {
               "inputs": { },
               "outputs": { }
           }
       }
```
are interpreted in the same way